client/my-sites/hosting/main.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+const Hosting = () => <p>Placeholder</p>;
```

Comment:
We can leave this for follow up, but we'll likely need to use the Main component from `'components/main'` and a `PageViewTracker` for basic page view analytics